src/pages/get-started/installation.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Install Adobe I/O Events for Adobe Commerce
+description: Learn how to install the Commerce modules needed for Adobe I/O Events for Adobe Commerce.
+---
+
+# Install and configure Adobe I/O Events for Adobe Commerce
+
+Adobe I/O Events for Adobe Commerce requires Adobe Commerce 2.4.5 or later. Magento Open Source is not supported.
+
+After you have created an [App Builder project](./project-setup.md), you must install the Commerce modules that enable integrations with Adobe I/O Events.
+
+## Install Adobe I/O modules on Commerce
+
+The following steps apply to both Adobe Commerce on cloud infrastructure and on-premises installations. Cloud customers must perform additional steps to configure the `ece-tools` package.
+
+1. If you are running Commerce 2.4.5, use the following command to load the eventing modules:
+
+   ```bash
+   composer require magento/commerce-eventing=^1.0
+   ```
+
+   Commerce 2.4.6 beta and later loads these modules automatically.
+
+1. Update the project dependencies.
+
+  ```bash
+  composer update
```

Comment:
Composer update will be performed on the previous step unless you add `--no-update` option.
It makes sense to update the command in the previous step to 
```
composer require magento/commerce-eventing=^1.0 --no-update
```